File: public/programs/java/p9.java
import java.util.*;
class p9
{
	public static void main(String args[])
	{
		int a,count=0;
		Scanner scan=new Scanner(System.in);
		System.out.print("Enter a number: ");
		a=scan.nextInt();
		for(int i=2;i<=a/2;i++)
		{
			if(a%i==0)
			{
				System.out.println(a+" is not a prime number");
				count=1;
				break;
			}

		}
		if(count==0)
		{
			System.out.println(a+" is a prime number");
		}
    }

}

File: public/programs/c/p9.c
// Prime number or not

#include<stdio.h>

int main()
{
	int i,n,flag;
	printf("Enter the number");
	scanf("%d",&n);
	
	for(i=2;i<=n/2;i++)
	{
		if(n%i == 0)                       //checks whether n is divisible by other number
		{
			flag=1;
			break;                        // goes outsidee first for loop
		}
	}
	if(flag==1)
	printf("%d is not a prime number",n);
    else
	printf("%d is a prime number",n);
}

		
			
			
		